data/migrations/20190211203949_notes.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function (knex, Promise) {
+    return knex.schema.createTable('notes', notes => {
+        notes.increments('id')
+        notes.string('title', 128).notNullable
```

Comment:
Good idea adding a shorter title length here 👍 Odds are a note title will not need the default length of 255.